haikudepotserver-webapp/src/main/webapp/WEB-INF/includes/unsupported.jsp

```diff
@@ -47,6 +47,17 @@ environment is working.
             K dispoz&#xED;cii je aj
             <a href="/__multipage">zjednodu&#x161;en&#xE9; pou&#x17E;&#xED;vate&#x13E;sk&#xE9; rozhranie</a>.
         </div>
+        
+        <div class="unsupported-message-container">
+            Esta aplicação não funciona de forma adequada no seu navegador atual.
```

Comment:
Hi; is it possible to get those escaped characters represented by HTML entities?

Reply:
Sorry! I have just noticed that in my PR let that sentence with special characters. I was trying to follow the kind of pattern I had seen above. Just to clarify, do you prefer like this:

```suggestion
            Esta aplica&#xE7;&#xE3;o n&#xE3;o funciona de forma adequada no seu navegador atual.
```

...or like this?:

```suggestion
            Esta aplica&ccedil;&atilde;o n&atilde;o funciona de forma adequada no seu navegador atual.
```

Reply:
Hello Victor; Probably the safest is the former variant such as ```&#xE7;```.  This ```unsupported.jsp``` page will appear on browsers that are very basic so they may not support the full range of HTML entities.